docs/catalog.rst

```diff
@@ -1,5 +1,41 @@
 .. _sec_catalog:
 
+With ``stdpopsim``, you can run simulations from a number of bespoke demographic models
+that were chosen from notable manuscripts. These models have been rigorously checked 
+and tested by multiple people, so you can rest easy knowing that your simulations are
+reproducible and bug-free!
+
+This catalog shows you all of the possible bespoke simulations that
+you can configure and run with ``stdpopsim``.
+It is organised around a number of choices that you'll need to make:
+
+1. Which **chromosome**? (ie. which :class:`.Genome` object?)
+2. Which **genetic map**? (ie. which :class:`.GeneticMap` object?)
+3. Which **model of demographic history**? (ie. which :class:`.Model` object)
+
+For instance, suppose you are interested in simulating human samples of
+
+1. chromosome 22, using
+2. the HapMapII genetic map, under
+3. a 3-population Out-of-Africa model.
+
+The following command simulates 2 samples from each of the three populations,
+and saves the output to a file called ``test.trees``:
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o test.trees -g HapmapII_GRCh37  --model ooa_3 2 2 2
+
+(To learn more about using ``stdpopsim`` via the command-line, read our tutorial
+about it.)
+
+.. todo::
+	Add a link to the CLI tutorial when it's finished.
+
+Are there other well-known organisms, genetic maps or models that
+you'd like to see in ``stdpopsim``? Head to our :ref:`sec_development`
```

Comment:
I guess this is kindof up to the consortium, but I'd be hesitant to use the issue page as a place where users just dump lots of requests for models. I think it would be better to encourage them to read the development page first, so they understand the process (and labour) that is needed to get a new model into stdpopsim, and then link them to the Issues page somewhere in the developers instructions. I've edited the text in a way that hopefully incorporates all of these perspectives, lmk what you think.

Reply:
Fair enough. I agree, let's see what @popgensims/all think.

Reply:
I like @gtsambos version --- we're trying to recruit people as much as anything

Reply:
okay, I'll leave it as is.